apps/jamtools/server/src/trace-function.ts

```diff
@@ -0,0 +1,46 @@
+import { trace } from '@opentelemetry/api';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function traceFunction<T extends (...args: unknown[]) => any>(functionName: string, fn: T): T {
+    return function (...args: Parameters<T>): ReturnType<T> | Promise<ReturnType<T>> {
+        const tracer = trace.getTracer('default');
+        let jsonArgs = '';
+        try {
+            jsonArgs = JSON.stringify(args);
+            // eslint-disable-next-line no-empty
+        } catch (e) {}
```

Comment:
Haven't figured out how to see this in SigNoz. Probably good to just get rid of this anyway